packages/expo-image-manipulator/android/src/main/java/expo/modules/imagemanipulator/FileUtils.kt

```diff
@@ -0,0 +1,31 @@
+package expo.modules.imagemanipulator
+
+import android.content.Context
+import android.graphics.Bitmap.CompressFormat
+import java.io.File
+import java.io.IOException
+import java.util.*
+
+internal object FileUtils {
+  @Throws(IOException::class)
+  fun generateRandomOutputPath(context: Context, compressFormat: CompressFormat): String {
+    val directory = File(context.cacheDir.toString() + File.separator + "ImageManipulator")
+    return directory.toString() + File.separator + UUID.randomUUID().toString() + toExtension(compressFormat)
```

Comment:
```suggestion
    val directory = File("${context.cacheDir.toString()}${File.separator}ImageManipulator")
    return "${directory.toString()}${File.separator}${UUID.randomUUID().toString()}${toExtension(compressFormat)
}"
```
I'm not sure if this is the best place to use string interpolation 🤷🏼‍♂️